intellij-ext/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecDialog.kt

```diff
@@ -16,9 +16,19 @@ import javax.swing.event.DocumentListener
 object MirrordExecDialog {
     private const val dialogHeading: String = "mirrord"
     private const val targetLabel = "Select Target"
+    private var lastSelectedTarget: String? = null
```

Comment:
Why do we need to store it here and in the settings too?

Reply:
trying to keep the behavior same as what was done here https://github.com/metalbear-co/mirrord/pull/1349/files we keep it for both global and workspace and I'm trying the same by mapping it persistent storage for global

Reply:
you mean in case of intellij? the one stored in the object variable will be lost when you reopen the workspace

Reply:
The difference between the object and the setting is just that one is persistent, so why not always use the persistent one?